doc/sphinx/addendum/extraction.rst

```diff
@@ -431,11 +431,12 @@ Additional settings
 
 .. opt:: Extraction File Comment @string
 
-   Provides a comment that is included at the beginning of the output files.
+   This :term:`option` can be used to provide a comment that is
```

Comment:
```suggestion
   This :term:`option` provides a comment that is
```

Reply:
My reason to use a more verbose wording was to make clear that it is the user who provides the comment (through the option). If you tell me that it is just as clear with your suggestion, then I'm happy to apply it.

Reply:
I think the extra words are superfluous because they could be applied to every option, flag, tactic and command.  These things exist specifically to cause Coq to do something--it seems quite obvious and thus distracting/unnecessary.  Indeed, these words could be applied to virtually any user-accessible feature of any software ("The Enter key _can be used to_ cause the system to process your command.")